p2p/dht/findnode_test.go

```diff
@@ -79,3 +80,16 @@ func TestFindNodeProtocol_FindNode2(t *testing.T) {
 	assert.NoError(t, err, "Should not return error")
 	assert.Equal(t, expected, idarr, "Should be array that contains the node")
 }
+
+func Test_ToNodeInfo(t *testing.T) {
```

Comment:
this is just an extra test, unrelated to the PR. but I kept it since it helped me check things out in the PR.